modules/claims_api/app/controllers/concerns/claims_api/poa_verification.rb

```diff
@@ -115,6 +99,47 @@ def verify_power_of_attorney!
       def poa_code_in_organization?(poa_code)
         ::Veteran::Service::Organization.find_by(poa: poa_code).present?
       end
+
+      private
+
+      def exactly_one_rep_match?(reps, poa_code)
+        reps.first.poa_codes.include?(poa_code) if reps.count == 1
+      end
+
+      def find_by_suffix(poa_code)
+        return false if @current_user.suffix.blank?
+
+        last_name_with_suffix = "#{@current_user.last_name} #{@current_user.suffix}"
```

Comment:
originally thought something like combo'ing these values in an array and doing a compact blank, but since the last name search goes first, worst case, this would just run last name plus trailing space which could actually be in the DB as we've seen from before.... so this actually may be a better fallback check.

Reply:
> since the last name search goes first, worst case, this would just run last name plus trailing space which could actually be in the DB as we've seen from before.... so this actually may be a better fallback check.

Interesting use case! Unfortunately, I donʼt think we run this check unless the current_user has a suffix. But if we want to try last_name + space, we can remove the guard at the top of this function.

Reply:
ahh, I missed line 110. I wonder if it's worth letting it run? worst case we search that last name + space and it just returns nothing again? I'll let you make that call. I believe we made a change to strip empty spaces at the end of the last name on an import, so we shouldn't see it any longer, and the DB should have correct names now, but ? there may be some older ones that are persisting that format as we haven't cleaned up yet.